typings/jsonld.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'jsonld'
```

Comment:
Can you provide typings for the parts of jsonld being used in this PR?

Reply:
Can you provide some guidance? Point me to an example?

Here's the context:

I’m using two functions from jsonld: `expand` and `canonize` (so far). Eventually the plan is to also add `sign` and `verify`, what do I need to define in the type file for jsonld (since it isn’t typescript)?

`canonize` returns a string form of the document (edited)
`expand` returns an object in which all of the keys have been expanded to the url definition of each attribute of the document, per the provided context.

e.g.
  ```{
    'http://schema.org/CreativeWork': [
      {
        'http://schema.org/author': [
          {
            '@value': 'Edgar Allan Poe',
          },
        ],
        'http://schema.org/dateCreated': [
          {
            '@value': '',
          },
        ],
        'http://schema.org/datePublished': [
          {
            '@value': '1845-01-29T03:00:00.000Z',
          },
        ],
        'http://schema.org/name': [
          {
            '@value': 'The Raven',
          },
        ],
        'http://schema.org/keyword': [
          {
            '@value': 'poem',
          },
        ],
        'http://schema.org/text': [
          {
            '@value': 'Once upon a midnight dreary...',
          },
        ],
      },
    ],
    'http://purl.org/dcterms/created': [
      {
        '@value': 'Mon Nov 13 2017 15:00:00 GMT+0000 (UTC)',
      },
    ],
    'http://schema.org/Text': [
      {
        '@value': '02badf4650ba545608242c2d303d587cf4f778ae3cf2b3ef99fbda37555a400fd2',
      },
    ],
    'http://schema.org/additionalType': [
      {
        '@value': 'Work',
      },
    ],
  },
]
```

Reply:
I think @jwicks31 and @kennylavender have experience adding typings for libraries not written in TypeScript (I believe that's how we added support for the feature toggle libraries). Maybe one of them can help?